app/views/application_drafts/_student_fields.html.slim

```diff
@@ -9,17 +9,15 @@
+  - unless in_same_team_as_other_student?(student)
+    = s.input :application_gender_identification, as: :string, label: 'Your gender self-identification', input_html: { disabled: !may_edit?(s.object), "data-genders-src" => ApplicationDraft::GENDER_LIST.to_json, class: "gender-form-field"}, wrapper_html: { class: student_field_error_class(index, application_draft, :application_gender_identification) }, hint: (student_field_error_hint(index, application_draft, :application_gender_identification) ||  'No-one is excluded from applying on the basis of gender, but people who self-identify as female, or have experiences being socialized as female, are given preference during selection.')
+  = s.input :application_diversity, as: :character_limited, label: 'Tell us about any special circumstances which you want us to consider while reviewing your application', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_diversity) }, hint: (student_field_error_hint(index, application_draft, :application_diversity) || 'E.g., LGBTQI, people of color, people with disabilities, single parents, people taking care of family members, etc.')
   = s.input :application_location, as: :city, label: 'Where do you plan to live July-September? (Please write as "City, Country")', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_location) }, hint: student_field_error_hint(index, application_draft, :application_location)
 
 
-  hr
-  .alert.alert-warning Please don't mention any specific names (e.g., geographical features, universities, companies, etc.) that may point to your location in any of the following fields. This will avoid bias in the selection process.
-
   h2 Background Information
 
-  = s.input :application_about, as: :character_limited, label: 'Please tell us a little bit about yourself. What is your background?', input_html: { rows: 5, disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_about)}, hint: (student_field_error_hint(index, application_draft, :application_about) || 'Only share what you feel comfortable with.')
+  = s.input :application_about, as: :character_limited, label: "Please tell us a little bit about yourself. What is your background?<br>If you're a university student, please tell us what courses you'll be taking at any time during the summer, as well as how many weekly hours these take up.".html_safe, input_html: { rows: 5, disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_about)}, hint: (student_field_error_hint(index, application_draft, :application_about) || 'Only share what you feel comfortable with.')
```

Comment:
Can you please move these changes to the label into the `config/locales/views.en.yml` file I created in #765 when rebasing this on `master`?